### PR TITLE
Add Konstellation Network and its native DARC token.

### PR DESCRIPTION
**Pull request by Yurist-85**

### CODE REVIEW

File: konstellation/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                {
+                    "denom": "udarc",
+                    "exponent": 0
+                },
+                {
+                    "denom": "darc",
+                    "exponent": 6
+                }
+            ],
+            "base": "darc",
```
Comment (sunnya97) [inline konstellation/assetlist.json:17]: Are you sure this shouldn't be udarc?

**Comment by sunnya97** [comment]
Can you add a logo to the assetlist

File: konstellation/assetlist.json (hunk 1 above)
Comment (Yurist-85) [reply konstellation/assetlist.json:17]: Yep, It should be :) Thank you. I'll fix it.

**Comment by Yurist-85** [comment]
Added SVG logo to asset list.
Fix base. Updated mainnet genesis location.

**Pull request merged**